### DISCUSSION TITLE
How to search for a number phrase in site title

### DISCUSSION THREAD

**Discussion by Wernbe** [microsoft-search/pnp-modern-search#3102] [Q&A] [answered]
Hi there,

Hopefully somebody know how to search for a number in the title of a site.  For example the title of the site is "Project101 search for projectnumber". When searching for 101 we dont get any result.

In the pnp search result webpart i have specified the following:

Title:{SearchTerms} AND
contentclass="STS_Site"

Thank you for your time!

**Comment by kasperbolarsen** [comment, marked as answer]
Hi, the short version is that you can't search for "ends with", only "begins with".

However there are ways to circumvent this limitation :-) :
(Title:{SearchTerms} OR Title:Project{SearchTerms}) AND
contentclass="STS_Site"

Sharing is caring :-)

**Comment by Wernbe** [comment]
Hi Kasper, thank you!



From: Kasper Larsen ***@***.***>
Date: Wednesday, 28 June 2023 at 15:16
To: microsoft-search/pnp-modern-search ***@***.***>
Cc: Werner Been ***@***.***>, Author ***@***.***>
Subject: Re: [microsoft-search/pnp-modern-search] How to search for a number phrase in site title (Discussion #3102)

Hi, the short version is that you can't search for "ends with", only "begins with".

However there are ways to circumvent this limitation :-) :
(Title:{SearchTerms} OR Title:Project{SearchTerms}) AND
contentclass="STS_Site"

Sharing is caring :-)

**Comment by kasperbolarsen** [comment]
Did it solve your problem? :-)

**Comment by Wernbe** [comment]
Hi Kasper,

It still not possible to search for a projectnumber if it partial. For example Project00001234 is the site title. When searching for 234 is doesn’t show the project. It works if I search for 1234 and I use this:

(Title:{SearchTerms} OR Title:Project0000{SearchTerms}) AND
contentclass="STS_Site"

But from what i understand from you it doesnt get better than this.

Thank you for your time.

Greetings,

Wern


From: Kasper Larsen ***@***.***>
Date: Thursday, 29 June 2023 at 08:51
To: microsoft-search/pnp-modern-search ***@***.***>
Cc: Werner Been ***@***.***>, Author ***@***.***>
Subject: Re: [microsoft-search/pnp-modern-search] How to search for a number phrase in site title (Discussion #3102)

Did it solve your problem? :-)
